docs/topics/api/addons.rst

```diff
@@ -492,6 +492,7 @@ Four recommendations are fetched, but only valid, publicly available addons are
 
 .. http:get:: /api/v5/addons/recommendations/
 
+    :query string app: Set the :ref:`add-on application <addon-detail-application>` for that query. This won't filter the results. Defaults to `firefox`.
```

Comment:
It default to `firefox` if firefox compatible but presumably it'd default to `android` if it wasn't?  Meh, I suppose it's not that important.

Reply:
``` ``firefox`` ```<- double back tick for literals.